fallout/fishing/fishing.dm

```diff
@@ -0,0 +1,86 @@
+// Actual fishing, this will usher in a new era for Fallout13
+
+// Fishing tiles
+
+/turf/open/floor/plating/ice/hole
+	name = "ice hole"
+	baseturfs = /turf/open/floor/plating/asteroid
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "asteroid"
+	icon_plating = "asteroid"
+	footstep = FOOTSTEP_SAND
+	barefootstep = FOOTSTEP_SAND
+	clawfootstep = FOOTSTEP_SAND
+	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	var/list/fish = list(/obj/item/reagent_containers/food/snacks/meat/slab/fish/sockeye,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/smallmouth,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/largemouth,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/lamprey,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/pink,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/chum,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/sturgeon,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/asian,
+		/obj/item/reagent_containers/food/snacks/meat/slab/fish/blinky)
+	var/fished = FALSE
+
+/turf/open/floor/plating/ice/hole/attackby(obj/item/W, mob/user, params)
```

Comment:
```suggestion
/turf/open/floor/plating/ice/hole/attackby(obj/item/W, mob/user, params)
	. = ..()
	if(!.)
		if(istype(W, /obj/item/fishing_rod))
			var/obj/item/fishing_rod/the_rod = W
			if(!can_fish(user))
				return TRUE

			if(!isturf(user.loc))
				return

			to_chat(user, "<span class='notice'>You start fishing...</span>")

			if(do_after(user, the_rod.fish_speed))
				to_chat(user, "<span class='notice'>You reel in your catch.</span>")
				getFished()
```

Reply:
Wasn't paying a ton of attention my bad

Reply:
I believe you don't really need isturf on a check for the user.loc considering that will always return true as the user will always be on a turf unless they are in nullspace in which case they won't be doing much of anything because the games unplayable for them at that point.